i18n/zh/docusaurus-plugin-content-docs/current/08-plugins/vault.md

```diff
@@ -0,0 +1,147 @@
+
+
+in the  command below, the variable $NAMESPACE you should replace with "hashicorp-vault" if you not modify the namespace variable.
+Otherwise, use the namespace name you replaced
+
+1.
+```
+   # Initialize vault-0 with one key share and one key threshold.
+   kubectl exec vault-0 -n $NAMESPACE -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+```
+2 .
+  ```
+   # Display the unseal key found in cluster-keys.json
+   cat cluster-keys.json | jq -r ".unseal_keys_b64[]"
+   ```
+3.
+```
+   # Create a variable named VAULT_UNSEAL_KEY to capture the Vault unseal key.
+   VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r ".unseal_keys_b64[]")
+```
+4.
+```
+   # Unseal vault-0 running on the vault-0 pod.
+   kubectl exec vault-0  -n $NAMESPACE -- vault operator unseal $VAULT_UNSEAL_KEY
+```
+you will see the above command's output like this, make sure the item "Initialized" value is "true" and the item "Sealed" value is "false"
+```shell
+Key                     Value
+---                     -----
+Seal Type               shamir
+Initialized             true
+Sealed                  false
+Total Shares            1
+Threshold               1
+Version                 1.9.2
+Storage Type            raft
+Cluster Name            vault-cluster-14052440
+Cluster ID              7630cd33-2ee1-39c1-db3f-e48a6d79970a
+HA Enabled              true
+HA Cluster              https://vault-0.vault-internal:8201
+HA Mode                 active
+Active Since            2022-04-23T16:45:47.6060163Z
+Raft Committed Index    30
+Raft Applied Index      30
+```
+
+5. Initialize vault-1 and vault-2 like vault-1
+
+```shell
+# Initialize vault-1
+kubectl exec vault-1 -n $NAMESPACE -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r ".unseal_keys_b64[]")
+kubectl exec vault-1  -n $NAMESPACE -- vault operator unseal $VAULT_UNSEAL_KEY
+# Initialize vault-2
+kubectl exec vault-1 -n $NAMESPACE -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r ".unseal_keys_b64[]")
+kubectl exec vault-1  -n $NAMESPACE -- vault operator unseal $VAULT_UNSEAL_KEY
+```
+
+6. 
+```
+   # Verify all the Vault pods are running and ready.
+   kubectl get pods -n $NAMESPACE
+```
+
+you will see the above command's output like this bellow, make sure all the pods are running and ready
+```
+NAME                                 READY   STATUS    RESTARTS   AGE
+vault-0                              1/1     Running   0          2m29s
+vault-1                              1/1     Running   0          2m29s
+vault-2                              1/1     Running   0          2m29s
+vault-agent-injector-68dc986-bnsj2   1/1     Running   0          2m28s
+
+```
+
+7. After the above operations, you want to use the vault to write/read secrets. 
+   You need to follow the documentation of hashicorp vault.
```

Comment:
Why are there two spaces before this line?

Please do a preview for this Markdown doc.

Reply:
> Why are there two spaces before this line?
> 
> Please do a preview for this Markdown doc.

Done.